src/routes/userRoutes.js

```diff
@@ -0,0 +1,48 @@
+import express from 'express';
+import userController from '../controllers/userController';
+import signupValidator from '../validation/signupValidation';
+
+const app = express.Router();
+
+/**
+ * @swagger
+ * /users/auth/signup:
+ *   post:
+ *     description: Signup
+ *     produces:
+ *      - application/json
+ *     parameters:
+ *      - in: body
+ *        name: user
+ *        description: The user to be created
+ *        schema:
+ *          type: object
+ *          required:
+ *            - username
+ *            - email
+ *            - password
+ *          properties:
+ *            username:
+ *              type: string
```

Comment:
Hi @lemoissonM I see redundant code in `userRoutes.js`

Reply:
i think it is related to swagger documentation, maybe @lemoissonM  will explain it to us

Reply:
@swaibat  I will send a link to writing documentation using swagger in the group in some seconds